msal/mock_http_manager.go

```diff
@@ -3,20 +3,20 @@
 
 package msal
 
-import "github.com/stretchr/testify/mock"
+import (
+	"net/http"
+
+	"github.com/stretchr/testify/mock"
+)
 
 type mockHTTPManager struct {
 	mock.Mock
 }
 
-//Get mocks the Get method of a HTTPManager
-func (mock *mockHTTPManager) Get(url string, requestHeaders map[string]string) (HTTPManagerResponse, error) {
-	args := mock.Called(url, requestHeaders)
-	return args.Get(0).(HTTPManagerResponse), args.Error(1)
-}
-
-//Post mocks the Post method of a HTTPManager
-func (mock *mockHTTPManager) Post(url string, body string, requestHeaders map[string]string) (HTTPManagerResponse, error) {
-	args := mock.Called(url, body, requestHeaders)
-	return args.Get(0).(HTTPManagerResponse), args.Error(1)
+func (m *mockHTTPManager) Do(req *http.Request) (*http.Response, error) {
```

Comment:
Can this file be removed given that `http_manager` is being removed?

Reply:
Not at present as this is still used as part of the mock testing.